README.md

```diff
@@ -1 +1,32 @@
-# robinson-group-python-template
\ No newline at end of file
+## Robinson Group Python Template
+This repo serves as a template for python projects in the [Robinson Group](https://robinsongroup.github.io/) of the BIH.
+To get started copy or fork this project and follow the steps below.
+### Install conda
+If you not already have conda, miniconda or Anaconda installed. Check out the [miniconda installation guide](https://www.anaconda.com/docs/getting-started/miniconda/install#quickstart-install-instructions).
+
+### Setup env
+Open `requirements/environment.yml` and adjust `<Project-Name>`.
+
+To create and activate the env execute:
+
+```
+conda env create -f requirements/environment.yml
+conda activate <Project-Name>
+```
+### Ruff Linting and formatting
+To automatically run ruff linting install the plugin of your favorite IDE.
+
+[Pycharm - Ruff Plugin](https://plugins.jetbrains.com/plugin/20574-ruff)
+
+[VSCode - Ruff Plugin](https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff)
+
+### Continuous Distribution
+You've done it, your package is at a state, where you want other to use it. Luckily, this repository features a CD-Pipeline,
+that will build and upload your packages to Pypi for you. To get the CD running you need to first follow these [instructions](https://docs.pypi.org/trusted-publishers/adding-a-publisher/). The pipeline uses [OIDC](https://openid.net/developers/how-connect-works/) to authenticate on Pypi. Make sure your [pyproject.toml](pyproject.toml) is set up correctly.
```

Comment:
I think an example would be nice here, to give the reader a gist of what needs to be tailored in their project to enable the CD.

I get that the instructions must be read first. However, please consider if adding a code snippet here (or in the `pyproject.toml`, FWIW) could be useful.

Reply:
I dont know what you mean.
You do not need to do anything, besides enabling trusted publishing on pypi and renaming stuff in the pyproject.toml.

Reply:
> ... and renaming stuff in the pyproject.toml.

this means renaming stuff that you would rename anyway, e.g. changing `robinson-group-python-template` or `project_name` to e.g. `gpsea`, right?

If enabling CD requires nothing else besides renaming obvious stuff and enabling trusted publishing, then this comment is resolved and we need no snippets here.